istqb-ct-genai-complete-guide.md

```
@@ -0,0 +1,1630 @@
+"""
+
+# ───── 実際の使用例 ─────
+
+actual_prompt = test_analysis_prompt.format(
+    project_description="Eコマースサイトのショッピングカート機能",
+    user_story="""
+    As a registered user,
+    I want to add items to my shopping cart,
+    So that I can purchase multiple items at once.
+
+    受入基準：
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)